Extending/Property-Editors/Property-Actions/index.md

```diff
@@ -0,0 +1,71 @@
+---
+versionFrom: 8.4.0
+meta.Title: "Umbraco Property Editors - Property Actions"
+meta.Description: "Guide on how to impelement Property Actions for Property Editors in Umbraco"
+---
+
+# Property Actions
+
+Property Actions is a build-in feature that provides a generic place for secondary functionality for property editors.
+Appearing as a small button next to the label of the property, that expands to show the available actions. Those actions are defined and implemented in the Property Editor, making it very open what a Property Action is.
+
+![Example of Property Action on Nested Content Property Editor](example-of-property-actions.jpg)
+
+## Data Structure of Property Actions
+Property Editors are a array of objects defining each action. 
+An action is defined by the following properties:
+
+```
+{
+    labelKey: 'clipboard_labelForRemoveAllEntries',
+    labelTokens: [],
+    icon: 'trash',
+    method: removeAllEntries,
+    isDisabled: true
+}
+```
+
+We use `labelKey` and `labelTokens` to retrieve a localized string that's begin displayed as the Actions label. [See localization for more info.](../../Extending/Language-Files/)
+`isDisabled` is used to disable an Action, which change the visual appearance and prevents interaction. Use this option when an action gives no impact. In the example above, it does not make sense to remove all entries if there are no entries.
```

Comment:
"Use this option when an action gives no impact." 
What exactly does this mean? 🤔